src/quo/components/counter/step/view.cljs

```diff
@@ -23,6 +24,22 @@
      [text/text
       {:weight :medium
        :size   :label
-       :style  {:color (style/text-color type theme)}} label]]))
+       :style  {:color (style/text-color type theme)}}
+      label]]))
 
-(def view (theme/with-theme view-internal))
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:accessibility-label {:optional true} [:maybe :keyword]]
+      [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+      [:in-blur-view? {:optional true} [:maybe :boolean]]
+      [:theme :schema.common/theme]
+      [:type {:optional true} [:enum :active :complete :neutral]]]]
+    [:value [:maybe [:or :string :int]]]]
+   :any])
+
+(def view
+  (quo.theme/with-theme
+   (schema/instrument #'view-internal ?schema)))
```

Comment:
FYI: This is the functional approach with `schema/instrument`. Notice the need to *var quote* `view-internal`. This has been added to the schema guidelines too.